lib/utils.zsh

```diff
@@ -65,6 +65,27 @@ spaceship::displaytime() {
   printf '%ds' $S
 }
 
+# Display seconds in human readable fromat, rounded to greater unit
+# USAGE:
+#   spaceship::displaytime_short <seconds>
+spaceship::displaytime_short() {
+    local T=$1
+    local D=$((T/60/60/24))
+    local H=$((T/60/60%24))
+    local M=$((T/60%60))
+    local S=$((T%60))
+
+    if [ $D -gt 0 ]; then
+        printf '%dd ' $D
+    elif [ $H -gt 0 ]; then
+        printf '%dh ' $H
+    elif [ $M -gt 0 ]; then
+        printf '%dm ' $M
+    else
+        printf '%ds ' $S
+    fi
```

Comment:
seems like the rest of the file uses 2 spaces for a tab, could you also change to 2 instead of 4?